tests/common-lib/clusterAccess.js

```diff
@@ -113,8 +95,8 @@ async function getUserContext(userName, ns) {
   try {
     t = execSync(`oc serviceaccounts get-token ${userName} -n ${ns}`)
   } catch (e) {
-    console.log('Failed to get service account token, will retry after 10 seconds.', e)
-    await sleep(10000)
+    console.log('Failed to get service account token, will retry after 9 seconds.', e)
+    await sleep(9000) // If this changes, must update timeout for tests using this function.
```

Comment:
Looking at the comment, can this be a global variable - so that we change in one place?

Reply:
Yes, I'll address in a separate PR because prow takes a while. 
The bigger issue is that I'm trying to recover from errors, but then the test takes longer and times out. I haven't found an easy way to handle the correlation between time to recover vs. test timeout.